src/common/replayBrowser/dao.ts

```diff
@@ -0,0 +1,183 @@
+import { FileResult } from "common/replayBrowser";
+import { app } from "electron";
+import path from "path";
+import * as sqlite3 from "sqlite3";
+
+const db = new sqlite3.Database(path.join(app.getPath("userData"), "statsdb.sqlite"), (err) => {
+  if (err) {
+    return console.error(err.message);
+  }
+  console.log("Connected to the replay database.");
+  db.run(`
+  CREATE TABLE IF NOT EXISTS replays (
+       fullPath      TEXT PRIMARY KEY,
+       name          TEXT,
+       folder        TEXT,
+       startTime     TEXT,
+       lastFrame     INTEGER,
+       playerCount   INTEGER,
+       player1       INTEGER,
+       player2       INTEGER,
+       player3       INTEGER,
+       player4       INTEGER,
+       settings      JSON,
+       metadata      JSON,
+       stats         JSON)
```

Comment:
Would kinda like to avoid having JSON as a column type. Would you consider having these as separate tables with typed/nullable information?